### PR TITLE
Separation of JavaScript files

### LINKED ISSUE #22: Seperate quoteHandle.js to two files
It might be helpful to consider using two separate files for quoteHandle.js instead of just one. Currently, we are checking the URL of the current page and utilizing the same file. However, due to the increasing file size, it would be beneficial to split it into two separate files.

### PR DESCRIPTION
**Pull request by Arisamiga**
This Pull Request separates the Javascript files so they are not all put together for easier expansion in the future. 

Closes #22

### CODE REVIEW

**Comment by Arisamiga** [comment]
@Alex23062007 Can you review this Pull Request so I can merge it?

**Comment by Alex23062007** [comment]
works fine to me. makes life much easier

File: script/optionHandle.js
Comment (Alex23062007) [inline script/optionHandle.js:1]: this might need popup function in the future

**Review by Alex23062007 — APPROVED** [review]
very nice. had to be done at some point

**Pull request merged**